src/components/ChatMessage.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+    IonCard,
+    IonCardContent,
+    IonCardHeader,
+    IonImg,
+    IonCardTitle,
+} from "@ionic/react";
+import "./ChatMessage.scss";
+
+interface ChatMessageProps {
+    sent?: boolean,
```

Comment:
Can we add some comments on what these properties mean? I'm a little unclear on what "sent" means, like does it mean its in the process of sending but its not delivered yet? Does it mean sent vs recieved?

Reply:
I'll get to that tonight.